### ISSUE TITLE
HTML tags will remove from source code when SourceEditing is enabled,

### ISSUE THREAD

**Issue by samanbahrekazemi** [ckeditor/ckeditor5#13556]
## 📝 Problem with **SourceEditing**

1. HTML tags will remove in source edit. Tags like iframe & video ...

**Comment by ryusei-48** [comment]
I am currently facing the same problem.
I am using CKEditor with React.
When I click on the "Source" button, it outputs the text with the html tags nicely deflected.
Yes, it behaves the same way as "textContent".
Can this bug be resolved?

**Comment by Witoso** [comment]
@ryusei-48 please be more precise about the problem you're experiencing. The OP most likely had an issue with tags that are not represented by any plugin. For this, usually the solution is [General HTML support](https://ckeditor.com/docs/ckeditor5/latest/features/html/general-html-support.html) plugin.

**Comment by ryusei-48** [comment]
Thank you for contacting us.
And sorry. I was able to resolve this issue myself, but forgot to contact you.
In my case, I was using CKEditor online builder to build the body and plugins. In doing so, the "Markdown output (https://ckeditor.com/docs/ckeditor5/latest/features/markdown.html)" plugin was included, so when I viewed the source code, there were no HTML tags! The cause of the problem was quite simple.
The cause of the problem was really quite simple.
I will continue to use this great editor.
